test/lib/output/ResultsFormatting.test.ts

```diff
@@ -10,9 +10,9 @@ import { PathlessEngineFilters, ENGINE, PMD6_VERSION, SFGE_VERSION } from '../..
 import { fail } from 'assert';
 import {Results, RunResults} from "../../../src/lib/output/Results";
 import { OutputFormat } from '../../../src/lib/output/OutputFormat';
-import {Controller} from "../../../lib/Controller";
-import {Pmd7CommandInfo, PmdCommandInfo} from "../../../lib/lib/pmd/PmdCommandInfo";
-import {PMD7_VERSION} from "../../../lib/Constants";
+import {Controller} from "../../../src/Controller";
```

Comment:
This is a simple fix-as-you-go item. This test was pointing to the lib directory instead of the src directory.